### ISSUE TITLE
Can the Chinese version of Guild Wars 2 client be used?

### ISSUE THREAD

**Issue by kof8855** [Healix/Gw2Launcher#275]
Can the Chinese version of Guild Wars 2 client be used?

**Comment by Healix** [comment]
Not really. Unlike the US version, the Chinese version doesn't support multiple clients sharing the same installation. If you try to play multiple accounts at the same time, the game is likely to crash.
